app/src/main/java/com/asfoundation/wallet/ui/iab/payments/common/error/IabErrorPresenter.kt

```diff
@@ -36,5 +47,27 @@ class IabErrorPresenter(
     )
   }
 
+  private fun handleSupportClick() {
+    disposables.add(Observable.merge(view.getSupportIconClicks(), view.getSupportLogoClicks())
+        .throttleFirst(50, TimeUnit.MILLISECONDS)
```

Comment:
Use observeOn(viewScheduler) due to throttleFirst running in computation scheduler. we already had crashes due to this on some devices